src/winrtble/peripheral.rs

```diff
@@ -143,26 +143,30 @@ impl Peripheral {
 
                 let mut local_name_guard = self.shared.local_name.write().unwrap();
                 *local_name_guard = Some(name.to_string());
+                drop(local_name_guard);
```

Comment:
This has no effect, the guard goes out of scope on the next line so would be dropped here anyway.